tests/phpunit/tests/ranks/update/2-4-0-alpha-4.php

```diff
@@ -201,7 +201,7 @@ public function test_update_user_ranks_table_duplicates() {
 				'user_id'    => (string) $user_id,
 				'rank_id'    => (string) $third_rank_id,
 				'rank_group' => 'points_type-points',
-				'blog_id'    => is_multisite() ? (string) $wpdb->blogid : '0',
+				'blog_id'    => is_multisite() ? (string) get_current_blog_id() : '0',
```

Comment:
This one also already uses the multisite conditional, so it is also good like you have made it now.